mos-platform/nes-unrom-512/_rompoke.ld

```diff
@@ -0,0 +1,2 @@
+/* Automatically generated by generate-rompoke-mapper-linkscript.lua. */
```

Comment:
Part of the generated output should be a human-readable description of the logic that this reduces to. This could just be hard-coded per target; that should also help make the intent for each target easier to read in the generator script.

Reply:
Should be taken care of likewise.